modules/transaction-payment/src/lib.rs

```diff
@@ -302,19 +334,38 @@ pub mod module {
 	#[pallet::getter(fn next_fee_multiplier)]
 	pub type NextFeeMultiplier<T: Config> = StorageValue<_, Multiplier, ValueQuery, DefaultFeeMultiplier>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn initial_balance)]
+	pub type PoolSize<T: Config> = StorageValue<_, Balance, ValueQuery>;
+
 	/// The alternative fee swap path of accounts.
 	#[pallet::storage]
 	#[pallet::getter(fn alternative_fee_swap_path)]
 	pub type AlternativeFeeSwapPath<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, BoundedVec<CurrencyId, T::TradingPathLimit>, OptionQuery>;
 
+	/// The ratio of foreign_asset_per_second to native_asset_per_second.
+	/// the reverse ratio means the value of foreign asset to native asset.
+	#[pallet::storage]
+	#[pallet::getter(fn fee_rate)]
+	pub type FeeRateOfToken<T: Config> = StorageMap<_, Twox64Concat, CurrencyId, Ratio, OptionQuery>;
```

Comment:
Please make sure the storage type name and getter name are consistent.

```suggestion
	#[pallet::getter(fn token_fixed_rate)]
	pub type TokenFixedRate<T: Config> = StorageMap<_, Twox64Concat, CurrencyId, Ratio, OptionQuery>;
```